src/ptp.rs

```diff
@@ -104,6 +65,83 @@ pub enum PtpResponseCode {
 
 }
 
+fn response_code_to_string(response: PtpResponseCode) -> String {
+    use ptp::PtpResponseCode::*;
+    match response {
```

Comment:
Or make these functions return `&'static str` because they don't need to allocate.